cursorless-talon-dev/src/cursorless_test.talon

```diff
@@ -9,8 +9,12 @@ test api command bring <user.cursorless_target>:
     user.cursorless_command("replaceWithTarget", cursorless_target)
 test api get text <user.cursorless_target>:
     user.cursorless_get_text(cursorless_target)
-test api get text list <user.cursorless_target>:
+test api get text list on <user.cursorless_target>:
```

Comment:
Why on?

Reply:
otherwise there's ambiguity with command on line above because "list" is a scope type

Reply:
I guess, but the tests were fine the way it was before. Unless we actually want to test with the list scope type I don't believe there is a problem. Anyhow I'm fine with being more verbose.

Reply:
Yeah I'm not sure why they started breaking for me tbh. But I think that it is better practice to make sure get text returns the right thing for the test anyway